internal/service/cluster/model_cluster.go

```diff
@@ -85,6 +85,10 @@ func flattenProcessArgs(p *matlas.ProcessArgs) []map[string]any {
 			"transaction_lifetime_limit_seconds":   p.TransactionLifetimeLimitSeconds,
 		},
 	}
+	if p.ChangeStreamOptionsPreAndPostImagesExpireAfterSeconds != nil {
+		flattenedProcessArgs[0]["change_stream_options_pre_and_post_images_expire_after_seconds"] = p.ChangeStreamOptionsPreAndPostImagesExpireAfterSeconds
```

Comment:
Curious does the API respond with a default value?

Reply:
No it doesn't, if it's not set the value returned by the API is null